addon-test-support/helpers.js

```diff
@@ -189,10 +189,10 @@ export function find(selector, contextEl) {
   } else if (Object.prototype.toString.call(selector) === "[object String]") {
     selected = document.querySelectorAll(`${settings.rootElement} ${selector}`);
   }
-  if (!settings.useJQueryWrapper) {
-    return (selected.length === 1) ? selected[0] : selected;
+  if (settings.useJQueryWrapper) {
+    return (!selected && selector.jquery) ? selector : Ember.$(selected);
```

Comment:
@rwjblue well for now we can back out of an option to wrap the find result with jQuery as developers can do that more explicitly, i.e. `$(find('.stuff'))`